src/core/public/chrome/ui/header/nav_link.tsx

```diff
@@ -125,13 +125,17 @@ export function createRecentNavLink(
   recentLink: ChromeRecentlyAccessedHistoryItem,
   navLinks: ChromeNavLink[],
   basePath: HttpStart['basePath'],
-  navigateToUrl: InternalApplicationStart['navigateToUrl']
+  navigateToUrl: InternalApplicationStart['navigateToUrl'],
+  workspaceEnabled: boolean = true
```

Comment:
The default value should be false I think?

Reply:
And I do not think the flag is needed here, if the recent item has workspaceId but the feature flag workspace is turned off, it should be be displayed to users.

Reply:
Indeed, it should be false.